packages/stops-overlay/package.json

```diff
@@ -18,9 +18,9 @@
     "url": "https://github.com/opentripplanner/otp-ui/issues"
   },
   "dependencies": {
-    "@opentripplanner/core-utils": "^3.0.4",
     "@opentripplanner/from-to-location-picker": "^1.0.3",
-    "@opentripplanner/zoom-based-markers": "^1.0.1"
+    "@opentripplanner/core-utils": "^4.1.0",
```

Comment:
What was the reason to change the a-z order of dependencies?

Reply:
Was a byproduct of only updating core-utils. Fixed in 11b29b2